ui/admin/tests/acceptance/auth-methods/update-test.js

```diff
@@ -359,31 +441,33 @@ module('Acceptance | auth-methods | update', function (hooks) {
     });
     await visit(urls.authMethods);
 
-    await click(`[href="${urls.authMethod}"]`);
-    await click(BUTTON_SELECTOR, 'Activate edit mode');
-    await fillIn(NAME_INPUT_SELECTOR, 'existing auth method');
-    await click(SAVE_BTN_SELECTOR);
+    await click(commonSelectors.HREF(urls.authMethod));
+    await click(commonSelectors.EDIT_BTN);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
+    await click(commonSelectors.SAVE_BTN);
     await a11yAudit();
 
-    assert.dom(ERROR_MSG_SELECTOR).hasText('The request was invalid.');
-    assert.dom('.hds-form-error__message').hasText('Name is required.');
+    assert.dom(commonSelectors.ALERT_TOAST_BODY).hasText(errorMessage);
+    assert.dom(commonSelectors.FIELD_NAME_ERROR).hasText(errorDescription);
   });
 
   test('saving an existing ldap auth method with invalid fields displays error messages', async function (assert) {
     featuresService.enable('ldap-auth-methods');
+    const errorMessage = 'The request was invalid.';
+    const errorDescription = 'scheme in url "" is not either ldap or ldaps';
```

Comment:
I know you didn't add this originally, but this reads incomplete/incorrect to me

Reply:
Sure, you mean the error description message? the error message or both?

Reply:
the description

Reply:
In other resources, we mention about a required field, such as name is required or worker_filter is required. But I don't think what we have for LDAP is right,  should we say "URL field is required?"

Reply:
@DhariniJeeva 

That is right, I just fix it. Thanks for the feedback.